apiserver/facades/agent/caasprovisioner/provisioner.go

```diff
@@ -34,11 +38,17 @@ func NewCAASProvisionerAPI(
 	if !authorizer.AuthController() {
 		return nil, common.ErrPerm
 	}
+	getAuthFunc := func() (common.AuthFunc, error) {
```

Comment:
the whole API requires controller, so maybe just pass AuthAlways into NewPasswordChanger?

Reply:
done